docs/installation/centos.md

```diff
@@ -12,153 +12,185 @@ parent = "smn_linux"
 
 Docker is supported on the following versions of CentOS:
 
-- [*CentOS 7 (64-bit)*](#installing-docker-centos-7)
-- [*CentOS 6.5 (64-bit)*](#installing-docker-centos-6.5) or later
+* CentOS 7.X 
+* CentOS 6.5 or higher 
+
+Installation on other binary compatible EL6/EL7 distributions such as Scientific
+Linux might succeed, but Docker does not test or support Docker on these
+distributions.
+
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using CentOS-managed packages, consult your
+CentOS documentation.
+
+## Prerequisites
+
+Docker requires a 64-bit installation regardless of your CentOS version. Also,
+your kernel must be 3.10 at minimum. CentOS 7 runs the 3.10 kernel, 6.5 does
+not. We make an exception for CentOS 6.5. To run Docker on
```

Comment:
@moxiegirl: 3 things:

1. I'm still not sure what the minimum kernel requirement is. I thought @unclejack explained previously that it is 3.8 although it has so many bugs that we strongly recommend 3.10+.
2. The wording is weird because it says "3.10 is minimum" but then 6.5 is an exception in another sentence. Maybe have it in one sentence: `Also, the minimum kernel requirement for CentOS 7 is 3.10, and for CentOS 6, it is 2.6.32-431`.
3. According to, https://github.com/docker/docker/pull/13887#issuecomment-111273242, I wonder if we should be refer to only CentOS 6, not 6.5. We could however say that 6.5 is the oldest supported version. /cc @vbatts @unclejack